.drone.star

```diff
@@ -28,8 +28,8 @@ PLUGINS_GITHUB_RELEASE = "plugins/github-release:1"
 PLUGINS_S3 = "plugins/s3"
 PLUGINS_S3_CACHE = "plugins/s3-cache:1"
 PLUGINS_SLACK = "plugins/slack:1"
-SELENIUM_STANDALONE_CHROME_DEBUG = "selenium/standalone-chrome-debug:3.141.59"
-SELENIUM_STANDALONE_FIREFOX_DEBUG = "selenium/standalone-firefox-debug:3.141.59"
+SELENIUM_STANDALONE_CHROME = "selenium/standalone-chrome"
+SELENIUM_STANDALONE_FIREFOX = "selenium/standalone-firefox"
```

Comment:
lets tag those images, so that nothing breaks in case of an update
```
SELENIUM_STANDALONE_CHROME = "selenium/standalone-chrome:104.0-20220812"
```